index.js

```diff
@@ -39,7 +40,14 @@ function validate(file, options, cb) {
       } else {
       	defaultReportFn(result.failed);
       }
-      cb();
+      if(options.emitError){
+        this.emit('error', new gutil.PluginError(pluginName, errorMessage));
+      }
+      if(options.errorInCallback){
+        cb(errorMessage);
+      }else{
+        cb();
+      }
```

Comment:
can you add spacing to make it consistent with rest of the file? Other than that, this looks good.